common/borders/borders-en.html

```diff
@@ -0,0 +1,125 @@
+				technologies.
+			</li>
+			<li>
+				Ensure that the border color has sufficient contrast with the
+				background color to ensure visibility for users with visual
+				impairments or color vision deficiencies.
+			</li>
+			<li>
+				Use additional visual cues, such as color, icons, or text, to
+				supplement the meaning conveyed by borders.
+			</li>
+			<li>
+				Responsive design should be considered, ensuring that borders
+				adapt well to different screen sizes and zoom levels.
+			</li>
+		</ul>
+
+		<h3 class="mrgn-tp-0 text-danger">
+			<span class="glyphicon glyphicon-remove-circle"></span> Incorrect
+			use
+		</h3>
+		<p>Compliance point(s):</p>
+		<ul>
+			<li>
+				Responsive design should be considered, ensuring that borders
+				adapt well to different screen sizes and zoom levels.
+			</li>
+			<li>
+				Don't rely on color alone to distinguish borders. Color-blind
+				users may have difficulty perceiving color differences, so use
+				additional visual cues like texture, patterns, or labels.
+			</li>
+			<li>
+				Don't use excessive or distracting borders that may overwhelm or
+				confuse users, especially those with cognitive disabilities or
+				attention disorders.
+			</li>
+			<li>
+				Don't use borders as a substitute for proper semantic structure
+				and organization. Use appropriate HTML elements, headings, and
+				other structural elements to convey content hierarchy.
+			</li>
+		</ul>
+	</div>
+	<div class="col-md-6">
+		<div class="panel panel-default">
+			<div class="panel-body">
+				<h3 class="mrgn-tp-0">Appearance</h3>
+				<p class="brdr-lft">
+					Example 1
+				</p>	
+				<p class="brdr-tp">
+                    Example 2
+				</p>
+				<p class="brdr-rght">
+                    Example 3
+				</p>
+			
+				<p class="brdr-bttm">
+                    Example 4
+				</p>
+			</div>
+		</div>
+        <div class="panel panel-default"">
+            <div class="panel-body">
+				<h3 class="mrgn-tp-0">Code</h3>
+                <pre><code>// Example 1:
+                    &lt;p <strong>class=&quot;brdr-lft&quot;</strong>&gt;...&lt;/p&gt;</code></pre>          
+				<pre><code>// Example 2:
+                    &lt;p <strong>class=&quot;brdr-tp&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+                <pre><code>// Example 3:
+                    &lt;p <strong>class=&quot;brdr-rght&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+      
+                <pre><code>// Example 4:
+                    &lt;p <strong>class=&quot;brdr-bttm&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+			</div>
+        </div>
+	</div>
+</div>
+
+<h3>CSS Class</h3>
+<dl class="dl-horizontal">
+	<dt><code>.brdr-left</code></dt>
+	<dd>Set a border to the left of the element</dd>
+	<dt><code>.brdr-tp</code></dt>
+	<dd>Set a border on top of the element</dd>
+	<dt><code>.brdr-rght</code></dt>
+	<dd>Set a border to the right of the element</dd>
+	<dt><code>.brdr-bttm</code></dt>
+	<dd>Set a border below the element</dd>
```

Comment:
You are missing the following classes: `.brdr-0` (to remove borders) and `.brdr-rds-0` (to remove border radius)